alea/examples/configs/unbinned_wimp_statistical_model_index_fitting.yaml

```diff
@@ -0,0 +1,79 @@
+parameter_definition:
+  wimp_mass:
+    nominal_value: 50
+    fittable: false
+    description: WIMP mass in GeV/c^2
+
+  livetime:
+    nominal_value: 2.
+    ptype: livetime
+    fittable: false
+    description: Livetime in years
+
+  wimp_rate_multiplier:
+    nominal_value: 1.0
+    ptype: rate
+    fittable: true
+    fit_limits:
+      - 0
+      - null
+    parameter_interval_bounds:
+      - 0
+      - 50
+
+  er_rate_multiplier:
+    nominal_value: 1.0
+    ptype: rate
+    uncertainty: 0.2
+    relative_uncertainty: true
+    fittable: true
+    fit_limits:
+      - 0
+      - null
+    fit_guess: 1.0
+
+  er_band_shift:
+    nominal_value: 0
+    ptype: shape_index
```

Comment:
Better name it as `index`.

Reply:
Then we have to assume `index` will always be the shape parameter?

Reply:
What is the other kind of index you will have?

Reply:
idk, if ppl think it's fine I am ok...